aggregator/pom.xml

```diff
@@ -31,6 +31,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-micrometer-registry-prometheus</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient_pushgateway</artifactId>
+            <version>0.12.0</version>
+        </dependency>
```

Comment:
Is the following dependency still required?
```
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-micrometer-registry-prometheus</artifactId>
        </dependency>
```

Reply:
I am able to create Gauges in the PushGateway without it, indeed. A bit unexpected as the simpleclient-pushgateway dependency doesn't contain micrometer-core dependency, but good to be able to remove it :)